lib/main.rb

```diff
@@ -1 +1,9 @@
-require_relative "tic_tac_toe"
\ No newline at end of file
+require_relative "tic_tac_toe"
+require_relative "print_the_board"
+
+#instatiate
+board = TicTacToe::Board.new
+display = TicTacToe::DisplayBoard.new(board.current_board)
```

Comment:
Consider separating getting the board `some_good_var_name = board.current_board` then using that when you instatiate `DisplayBoard`

Reply:
```
board = TicTacToe::Board.new
just_the_value_of_board = board.current_board
display = TicTacToe::DisplayBoard.new(just_the_value_of_board)
```

Reply:
Another thought - is `current_board` what we want this to be named? I mean because we already call the object `board`, and the `DisplayBoard` class has the name board in it as well, board is starting to lose all it's meaning - It's like we're saying "smurf" everywhere "smurf that smurfing smurftastic smurfage, you smurfing smurf!"

What do you think of calling `current_board` just `current`? It might read easier.